src/app/containers/Video/helpers/mediatorUrl.test.js

```diff
@@ -0,0 +1,32 @@
+import mediatorURL from './mediatorUrl';
+
+describe('mediatorURL', () => {
+  const tests = [
+    {
+      env: 'local',
+      expected: 'open.test.bbc.co.uk',
+      description: 'local env returns test url',
+    },
+    {
+      env: 'test',
+      expected: 'open.test.bbc.co.uk',
+      description: 'test env returns test url',
+    },
+    {
+      env: 'live',
+      expected: 'open.bbc.co.uk',
+      description: 'live env returns live url',
+    },
+    {
+      env: '',
+      expected: 'open.bbc.co.uk',
+      description: 'no env returns live url',
```

Comment:
```suggestion
      description: 'no env returns test url',
```

Reply:
No it's right as it is; was told if no env is specified we use live.